qemu/tests/pvpanic_event_check.py

```diff
@@ -53,6 +56,20 @@ def run(test, params, env):
             "within the timeout period" % kdump_expect_status
         )
 
+    error_context.context("Check ISA Bridge in the guest", test.log.info)
+    o = session.cmd_output(check_ISA_cmd)
+    device_id = o.split()[0]
+    device_cmd = device_cmd % device_id
+    o = session.cmd_output(device_cmd)
+    if o.strip() == "1" and ("i440fx" in machine_type or machine_type == "pc"):
```

Comment:
It could be controlled over configuration, something we defined well there to avoid such many `if...else` block.